### PR TITLE
Gt meet app2.0

### PR DESCRIPTION
**Pull request by mikewguo**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -51,7 +51,7 @@ android {
         }
         release {
             // Uncomment the following line for singing a test release build.
-            //signingConfig signingConfigs.debug
+            signingConfig signingConfigs.debug
```
Comment (turbozv) [inline android/app/build.gradle:54]: revert, we only sign release

File: images/fishmeet.png
Comment (turbozv) [inline images/fishmeet.png:1]: where is this file used?

File: ios/app/app.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1031,7 +1035,8 @@
 				OTHER_CPLUSPLUSFLAGS = "$(inherited)";
 				OTHER_LDFLAGS = (
 					"$(inherited)",
-					" ",
+					"-Wl",
+					"-ld_classic",
```
Comment (turbozv) [inline ios/app/app.xcodeproj/project.pbxproj:1039]: are these changes necessary?

File: ios/app/src/Base.lproj/LaunchScreen.xib [hunk 3]
```diff
@@ -1,8 +1,10 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="7702" systemVersion="14D136" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" colorMatched="YES">
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="22505" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" colorMatched="YES">
+    <device id="retina6_12" orientation="portrait" appearance="light"/>
     <dependencies>
         <deployment identifier="iOS"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="7701"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="22504"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
```
Comment (turbozv) [inline ios/app/src/Base.lproj/LaunchScreen.xib:7]: version 8 seems very old, latest is v15

File: ios/app/src/Launch Screen.storyboard [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (turbozv) [inline ios/app/src/Launch Screen.storyboard:1]: is this file change necessary? can we reuse existing LaunchScreen?

File: react/features/base/config/actions.ts [hunk 5]
```diff
@@ -139,6 +139,12 @@ export function setConfig(config: IConfig = {}) {
             config.bosh = bosh;
         }
 
+        //Gracetech
```
Comment (turbozv) [inline react/features/base/config/actions.ts:142]: Please fix build issues.
Since we'll never merge back to master, please remove such comment

File: react/features/base/i18n/i18next.ts [hunk 6]
```diff
@@ -48,7 +48,7 @@ export const TRANSLATION_LANGUAGES: Array<string> = Object.keys(TRANSLATION_LANG
  * @public
  * @type {string} The default language.
  */
-export const DEFAULT_LANGUAGE = 'en';
+export const DEFAULT_LANGUAGE = 'zhCN'; //'en';
```
Comment (turbozv) [inline react/features/base/i18n/i18next.ts:51]: is this change necessary?

File: react/features/base/ui/components/updateTheme.native.ts [hunk 7]
```diff
@@ -5,5 +5,8 @@
  * @returns {Object}
  */
 export default function updateTheme(theme: Object) {
+    //Gracetech -- to compact more buttons 
+    theme.spacing[2] = 2;
```
Comment (turbozv) [inline react/features/base/ui/components/updateTheme.native.ts:9]: this is very hacky, is there a better way?

File: android/app/build.gradle (hunk 1 above)
Comment (JeffYin24) [reply android/app/build.gradle:54]: reverted

File: ios/app/app.xcodeproj/project.pbxproj (hunk 2 above)
Comment (JeffYin24) [reply ios/app/app.xcodeproj/project.pbxproj:596]: this is to address the expo EAS build issues

File: ios/app/src/Base.lproj/LaunchScreen.xib (hunk 3 above)
Comment (JeffYin24) [reply ios/app/src/Base.lproj/LaunchScreen.xib:7]: this is generated by xcode.

File: ios/app/src/Images.xcassets/AppIconDebug.appiconset/Contents.json [hunk 8]
```diff
@@ -13,37 +13,37 @@
     {
       "size" : "29x29",
       "idiom" : "iphone",
-      "filename" : "AppIcon-29@2x.png",
```
[the remark this answers is not in the document]
Comment (JeffYin24) [reply ios/app/src/Images.xcassets/AppIconDebug.appiconset/Contents.json:16]: want to keep the original appicon just in case.

File: react/features/base/config/actions.ts (hunk 5 above)
Comment (JeffYin24) [reply react/features/base/config/actions.ts]: fixed

File: react/features/base/i18n/i18next.ts (hunk 6 above)
Comment (JeffYin24) [reply react/features/base/i18n/i18next.ts]: fixed

File: react/features/base/ui/components/updateTheme.native.ts (hunk 7 above)
Comment (JeffYin24) [reply react/features/base/ui/components/updateTheme.native.ts]: fixed

File: ios/app/src/Launch Screen.storyboard (hunk 4 above)
Comment (JeffYin24) [reply ios/app/src/Launch Screen.storyboard:1]: don't want to mess with the original launchscreen.
